modules/bvl_feedback/README.md

```diff
@@ -0,0 +1,15 @@
+# Behavioural Feedback
+
+
+## Configurations
+
+The Feedback module allows users to flag and comment on  behavioural database fields.
```

Comment:
The first paragraph is still the purpose, it's just the second that I was saying is a configuration